_docker/environments/drupal-production/rhd.settings.php

```diff
@@ -6,3 +6,6 @@
   $config['redhat_developers'] = $yml_settings;
 }
 
+$settings['hash_salt'] = 'xuAWpK0fmrZ6UGofFcP3lBkcmdpdumWMLqvCbnYjFY85OgRXYvEKPItJDH66vs4UpeYORQXLHQ';
+$settings['install_profile'] = 'standard';
+$config_directories['sync'] = 'sites/default/files/config_BbPlfGDu86LJqlRwhA9RbCf38VZMDijNF-owvfhuzVL73hk7BtWwy3kfIlqKLXeiSgTA-MHeVw/sync';
```

Comment:
Do we need this file? In the docker-compose.yml for `drupal-production` I see the following in the volumes definition for Drupal:

```
- /credentials/drupal/rhd.settings.php:/var/www/drupal/web/sites/default/rhd.settings.php
```

Which would suggest this file is not being used?